src/Workspaces/CSharp/Portable/Extensions/ArgumentSyntaxExtensions.cs

```diff
@@ -105,9 +105,17 @@ public static RefKind GetRefKind(this ArgumentSyntax argument)
             SemanticModel semanticModel,
             CancellationToken cancellationToken)
         {
+            TypeInfo typeInfo;
```

Comment:
This code looks nearly identical to the same added code a few files earlier.  Can this code not be shared?